recipe/meta.yaml

```
@@ -40,30 +40,42 @@ requirements:
     - wheel
   run:
     - python
-    - numpy >=1.23
+    - numpy >=1.24  # from dask(array)
```

```suggestion
    - numpy >=1.23.2
```
The dependency on dask is enough for dask-array.